### PR TITLE
12282017 Re:Start Update / Renewal Update

### PR DESCRIPTION
**Pull request by iamtonysoft**
Updates for today

### CODE REVIEW

File: tables/iRO/Restart/shuffle.txt [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+0127 0962
+0140 035F
 0178 014D
-0198 0008
 014D 014F
 014F 0815
 08C9 0A69
-09CE 07E4
+09CE 094C
+0A0D 8B00000A
```
Comment (sctnightcore) [inline tables/iRO/Restart/shuffle.txt]: `0A0D 0A0D ?  , 0A0D 0A8B ? `

**Comment by ropedope** [comment]
@iamtonysoft can you do renewal as well? Thanks in advance :)

**Comment by iamtonysoft** [comment]
@ropedope no im lazy i was just asked to do this PR...

File: tables/iRO/Restart/shuffle.txt (hunk 1 above)
Comment (iamtonysoft) [reply tables/iRO/Restart/shuffle.txt]: recvpackets.txt
8B00000A -1 4 1

shuffle.txt
0A0D 8B00000A

**Comment by iamtonysoft** [comment]
Renewal Update. Need to test i got no client.
https://github.com/OpenKore/openkore/pull/1561/commits/0e2abf0343bd4b109972f8045dfd1c653f755542

sry i am noob dunno how to make seperate PR.

**Comment by natsume1st** [comment]
@iamtonysoft it works dude thx :)

**Comment by alisonrag** [comment]
did the Renewall not changed the cryptkeys?

**Comment by sctnightcore** [comment]
@alisonrag  yes ,

**Comment by alisonrag** [comment]
in this pr he updated only the restart key
renewall key is missing

**Comment by iamtonysoft** [comment]
@alisonrag sry i revert i dunno how to make seperate PR this is the reference: https://github.com/OpenKore/openkore/commit/0e2abf0343bd4b109972f8045dfd1c653f755542

**Comment by arfelgabani** [comment]
Hi, would like to ask if there is an update with openkore for iRO restart server. it gets stuck in map server and cannot connect to the game info of my open kore is uploaded. please let me know if i need to download any other file.

**Comment by sctnightcore** [comment]
@arfelgabani  wait merge PR !

**Comment by arfelgabani** [comment]
noted sir thanks....

**Pull request closed without merging**